config/api.yml

```diff
@@ -1122,6 +1122,13 @@
       :get:
       - :name: read
         :identifier: floating_ip_show
+  :folders:
+    :description: Datacenters
+    :identifier: datacenter_controller
```

Comment:
should description be Folders ?  Datacenter.all.collect(&:name) is a subset than EmsFolder.all.collect(&:name)

Reply:
@abellotti yup 😆 I started out doing datacenters...didn't change it to folders